bin/deploy.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+PUBLISH=0
+CHEF_VER="latest"
```

Comment:
Removed the option from `build_image.sh` and made the `deploy_wrapper` get it and provide from the the template.